gateway_to_backend/README.md

```diff
@@ -265,9 +264,9 @@ to monitor this change._
 
 - **Response:**
 
-    > **topics:** gw-response/set_config/*\<gw-id\>/\<sink-id\>*
+    > **topics:** gw-response/get_gateway_info/*\<gw-id\>/\<sink-id\>*
     >
-    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoResp][message_GetGwInfoResp] and [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoResp][message_GetGwInfoResp], [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]
```

Comment:
My comment was probably unclear :-)

I was suggesting to remove all the second part: [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]

Point is that if you follow first link [GetGwInfoResp] you will see that one field is GatewayInfo described just above.

If we do it here, why not doing it everywhere alos? Like giving link to what and EventHeader format is for example.

Do you agree? It is just a suggestion ;-)

Reply:
It makes sense to remove it and since we now have the class hierarchy I don't see a need for it anymore.